tools/cldr-code/src/main/java/org/unicode/cldr/util/DateTimeFormats.java

```diff
@@ -157,6 +157,10 @@ enum MyOptions {
     /**
      * Set a CLDRFile and calendar. Must be done before calling addTable.
      *
+     * Note: currently, this method will initialize the datetime pattern in `dateFormatItem`
+     * by taking "default" values only. That is to say, any alternate values, patterns annotated
+     * with an attribute like `alt="ascii"` will be skipped over and not returned.
+     *
```

Comment:
```suggestion
     * Note: currently, this method will skip `alt="ascii"` elements of `dateFormatItem`.
     * This may be configurable in the future. See CLDR-18580
     *
```